### PR TITLE
add info

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10070

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-new-install.md [hunk 1]
```diff
@@ -55,15 +55,17 @@ Windows Hello for Business must have a public key infrastructure regardless of t
 
 This guide assumes most enterprises have an existing public key infrastructure.  Windows Hello for Business depends on a Windows enterprise public key infrastructure running the Active Directory Certificate Services role from Windows Server 2012 or later.
 
+More details about configuring a Windows enterprise public key infrastructure and installing Active Directory Certificate Services can be found [here](https://docs.microsoft.com/en-us/windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-pki#follow-the-windows-hello-for-business-hybrid-key-trust-deployment-guide) and [here](https://docs.microsoft.com/en-us/windows-server/networking/core-network-guide/cncg/server-certs/install-the-certification-authority).
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-new-install.md:58]:
```suggestion
For more details about configuring a Windows enterprise public key infrastructure and installing Active Directory Certificate Services, see [Follow the Windows Hello for Business hybrid key trust deployment guide](/windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-pki#follow-the-windows-hello-for-business-hybrid-key-trust-deployment-guide) and [Install the Certification Authority](/windows-server/networking/core-network-guide/cncg/server-certs/install-the-certification-authority).
```

Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-new-install.md]:
```suggestion
For more details about configuring a Windows enterprise public key infrastructure and installing Active Directory Certificate Services, see [Follow the Windows Hello for Business hybrid key trust deployment guide](/windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-pki#follow-the-windows-hello-for-business-hybrid-key-trust-deployment-guide) and [Install the Certification Authority](/windows-server/networking/core-network-guide/cncg/server-certs/install-the-certification-authority).
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-new-install.md [hunk 2]
```diff
@@ -55,15 +55,17 @@ Windows Hello for Business must have a public key infrastructure regardless of t
...
+
+>[!NOTE]
+>Never install a certificate authority on a domain controller in a production environment.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-cert-new-install.md:61]:
```suggestion
> [!NOTE]
> Never install a certificate authority on a domain controller in a production environment.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-cert-new-install.md (hunk 1 above)
Comment (VLG17) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-cert-new-install.md]: @JohanFreelancer9 Do you see this change commited? I clicked on "Commit suggestion", nothing happened and now it's grayed out.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**